README.md

```diff
@@ -1,11 +1,16 @@
-# zoe ![Build](https://github.com/NudelErde/zoe/workflows/Build%20(with%20CMake)/badge.svg) [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FNudelErde%2Fzoe.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2FNudelErde%2Fzoe?ref=badge_shield)
-*zoe* is a video game engine that I am working on in my free time. It probably won't surpass *Unity3D* or *Unreal Engine* but where is the fun in using any of these if you can make your own!
+<h1 align="center">
+  Zoe
+</h1>
 
-I want it to be as optimized as possible but am limited to my free time.
+<p align="center">
+  Zoe is a video game engine that I am working on in my free time. It probably won't surpass <i>Unity3D</i> or <i>Unreal Engine</i> but where is the fun in using any of these if you can make your own! <br>I want it to be as optimized as possible but am limited to my free time.
```

Comment:
Would personally preferede Markdown instead of `<i></i>` @NudelErde?

Reply:
Markdown > html